### PR TITLE
feat(route): 东华大学 教务处-通知公告 and 信息公开网-最新公开信息

### PR DESCRIPTION
**Pull request by kirakiseki**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/dhu/jiaowu/news/student
/dhu/jiaowu/news/teacher
/dhu/xxgk/news
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [ ] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — CHANGES_REQUESTED** [review]
Please use v2 script standard https://github.com/DIYgod/RSSHub/discussions/8236

File: docs/university.md [hunk 1]
```diff
@@ -657,6 +657,22 @@ xskb1 对应 <http://www.auto.uestc.edu.cn/index/xskb1.htm>
...
+
+| 学生专栏 | 教师专栏 |
+| -------- | -------- |
+| student  | teacher  |
+
+</Route>
+
+### 最新信息公开
+
+<Route author="KiraKiseki" example="/dhu/xxgk/news" path="/dhu/xxgk/news">
```
Comment (DIYgod) [inline docs/university.md:674]: Missing closure label `</Route>`

**Review by DIYgod — CHANGES_REQUESTED** [review]

File: docs/university.md (hunk 1 above)
Comment (kirakiseki) [reply docs/university.md:674]: Is it still necessary for a route without any argument (like Line 674) to add a `</Route>` label? 
Is it similar to `docs/university.md` in Line 11?

Comment (TonyRL) [reply docs/university.md:674]: You need to close `<Route>` tag. It’s clearly stated [in the documentation](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-tian-jia-jiao-ben-wen-dang). 
In line 11 <Route> tag is closed properly but yours don’t.

Comment (kirakiseki) [reply docs/university.md]: Apologized for my carelessness.
Corrected by using `/>`

File: lib/v2/dhu/xxgk/news.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const got = require('@/utils/got');
+const iconv = require('iconv-lite');
+const cheerio = require('cheerio');
+const resolve_url = require('url').resolve;
+
+const base_url = 'https://xxgk.dhu.edu.cn/1737/list.htm';
+
+module.exports = async (ctx) => {
+    const link = `${base_url}`;
```
Comment (TonyRL) [inline lib/v2/dhu/xxgk/news.js:9]:
```suggestion
    const link = base_url;
```
Please fix lint problem

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Pull request merged**